README.md

```diff
@@ -1,2 +1,27 @@
 # adapt-federated-search
 Federated Search for SAA using Algolia
+
+
+## Embeddable Search Example
+This repo also contains a working example showing how a React-based search widget could be exported
+in a standalone form that could be embedded and used on other websites. 
+
+This is done with a new `export` script has been added to package.json. This will run webpack and generate a special bundle with just the `<Search>` component and stylesheet. 
+
+Here's some additional info on how this works:
+1. The webpack configuration for this build process can be found in [export/webpack.config.js](export/webpack.config.js). This is fairly simple as far as webpack configs go, but a couple key things to note are:
+    - MiniCSSExtract plugin is used to put CSS styles in a separate stylesheet.
+    - Babel is used with the `preset-react` to transpile the React code into browser-compatible Javascript.
+    - You may need to add additional loaders and plugins for your specific project.
+
+1. The entry script is [export/index.js](./export/index.js). This file just does a couple of things:
+    - Loads React.
+    - Imports the stylesheet.
+    - Grabs configuration parameters from the data attributes of the mounting element.
+    - Tells the script to mount the `<Search>` component in the DOM on the mounting element with the id of 'standalone-search'.
+
+1. To use the search component on another site, you will need to:
+    1. Run `npm run export` to run the build process.
+    1. Copy the contents of the `dist` folder to your other website.
```

Comment:
I'm hoping for one less step here. I would like to be able to embed the search feature on another site with just a script tag and possibly an HTML container from a hosted source. EG, I could tell someone to copy and paste the following code into their website.

```
<div 
  id="standalone-search" 
  data-app-id='your-aloglia-app-id' 
  data-api-key='your-algolia-search-api-key' 
  data-index-name='your-algolia-index-name'>
</div>
<script src="https://deploy-preview-1--adapt-federated-search.netlify.app/export/index.js"></script>
```